hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/hooks/useNewFields.tsx

```diff
@@ -42,19 +43,18 @@ type ReferenceInstancesProps = {
 const ReferenceInstances: FunctionComponent<ReferenceInstancesProps> = ({
     instances,
 }) => {
-    if (!instances || instances.length === 0) {
-        return <>{textPlaceholder}</>;
-    }
-
     return (
         <>
-            {instances.map((instance, index) => (
-                <Fragment key={instance.id}>
-                    {index > 0 && ', '}
-                    <span>
-                        <LinkToInstance instanceId={`${instance.id}`} />
-                    </span>
-                </Fragment>
+            {!instances || (instances.length === 0 && textPlaceholder)}
+            {instances.map(instance => (
+                <Box mb={1}>
+                    <InstanceDetail
+                        key={instance.id}
+                        instanceId={`${instance.id}`}
+                        minHeight="150px"
+                        titleVariant="subtitle1"
+                    />
+                </Box>
```

Comment:
```suggestion
                <Box mb={1} key={instance.id}>
                    <InstanceDetail
                        instanceId={`${instance.id}`}
                        minHeight="150px"
                        titleVariant="subtitle1"
                    />
                </Box>
```
There's a key error

Reply:
we can leave it like that, this is refactored on the other PR